test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -491,7 +491,6 @@ export class WebElementWrapper {
     return await this.retryCall(async function find(wrapper) {
       return wrapper._wrap(
         await wrapper._webElement.findElement(wrapper.By.css(testSubjSelector(selector))),
-        wrapper.By.css(selector)
```

Comment:
Note: Before anyone asks, no, replacing this line with `wrapper.By.css(testSubjSelector(selector))` would not work, because the selector would not include the parent's, therefor if there are multiple dom nodes with the same `dataTestSubj` value, any one would be returned (and the parent's selector can potentially be anything, not only a css selector, so concatenating them is not doable). 

Also FWIW, `findAllByTestSubject` does not pass a selector to the created instances.

Reply:
good catch, thank you!